src/Sentry.DiagnosticSource/Internals/DiagnosticSource/SentrySqlListener.cs

```diff
@@ -148,6 +171,11 @@ public void OnNext(KeyValuePair<string, object?> value)
                     TrySetConnectionStatistics(connectionSpan, value);
                     connectionSpan.Finish(SpanStatus.Ok);
                 }
+                else if ((value.Key is SqlMicrosoftWriteTransactionCommitAfter || value.Key is SqlDataWriteTransactionCommitAfter) &&
+                    GetSpan(SentrySqlSpanType.Connection, null, value.GetSubProperty<Guid>("Connection", "ClientConnectionId")) is { } connectionSpan2)
```

Comment:
Can this throw if props don't exist? `value.GetSubProperty<Guid>("Connection", "ClientConnectionId")`

Reply:
This is a safe call, if not found, it'll return the default value of T, in this case, an empty GUID.